scripts/populate_conferences.py

```diff
@@ -540,9 +540,18 @@ def main():
         'poster': True,
         'talk': True,
     },
+        'SMM2015': {
```

Comment:
Indent by 4--not 8-- spaces here (indentation is significant in Python).

I will fix this when I merge.